tfhe/src/core_crypto/gpu/mod.rs

```diff
@@ -60,58 +60,64 @@ impl CudaStream {
     }
 
     /// Allocates `elements` on the GPU asynchronously
-    pub fn malloc_async<T>(&self, elements: u32) -> CudaVec<T>
+    ///
+    /// # Safety
+    ///
+    /// - [CudaDevice::cuda_synchronize_device] __must__ be called after the copy
```

Comment:
cuda_synchronize_device or synchronize ? (does it matter ?)

Reply:
synchronize rather, we don't necessarily need to sync the whole GPU. I'll edit the comment.